config/application.rb

```diff
@@ -12,8 +12,7 @@
 # require "action_text/engine"
 require "action_view/railtie"
 # require "action_cable/engine"
-require "sprockets/railtie"
-# require "rails/test_unit/railtie"
+require "rails/test_unit/railtie"
```

Comment:
Why did this line change? 🤔

Has something changed such that it needs to be included now, where it wasn't needed before?

Reply:
Looks like it was uncommented by `rails app:update`.

I've commented the line and the tests run locally.  Will see if they're on CI.

Reply:
All looks ok in CI and locally.